Dockerfile

```diff
@@ -1,11 +1,103 @@
-FROM nfnty/arch-npm
+FROM opensuse:tumbleweed
```

Comment:
Still, the skipped and failing bear Tests mean that a lot of stuff is missing and in the end we might need to just use Ubuntu and a couple random PPAs anyway to get all packages